css/style.css

```diff
@@ -0,0 +1,64 @@
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto",
+    "Helvetica Neue", Arial, sans-serif;
+  /* font-size: 1.6rem; */
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+  margin: 0;
+  text-align: center;
+}
+
+#main {
+  background-color: #eee;
+  margin: 30px 10px;
+  padding: 15px;
+  
+}
+#title {
+  /* padding: 20px; */
+  margin-top: 24px;
+  font-size: 40px;
+
+}
+
+#img-div {
+  background-color: white;
+  padding: 10px;
+}
+#img-caption {
+  font-size: 16px;
+  margin: 15px;
+}
+
+ul {
+  max-width: 500px;
+  margin: 0 auto;
+}
+
+li {
+  margin: 15px 0;
+  text-align: left;
+  
+}
```

Comment:
While you did a good job implementing all the required comments :+1: please ensure to always state explicit reasons why you requested a change; it might be obvious to you and not so to others.